packages/@expo/cli/src/start/server/middleware/__tests__/resolvePlatform-test.ts

```diff
@@ -73,6 +74,45 @@ describe(parsePlatformHeader, () => {
   });
 });
 
+describe(resolvePlatformFromUserAgentHeader, () => {
+  it(`resolves ios from user-agent string`, () => {
+    expect(
+      resolvePlatformFromUserAgentHeader(
+        asRequest({
+          url: 'http://localhost:3000',
+          headers: {
+            'user-agent':
+              'Mozilla/5.0 (iPhone; CPU iPhone OS 15_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.2 Mobile/15E148 Safari/604.1',
```

Comment:
These user agent values need annotation about where they come from and how to update them in the future.